woocommerce/woocommerce
# fixed misaligned elements in generated refunds page issue

Fixed misaligned elements in generated refunds page issue.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/45213 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to WP Admin > Pages. If your site already has a "Refund and Returns Policy" page, delete it.
2. Using wp-cli, run `wp shell`. Then in the shell, run `WC_Install::create_pages()`. This will regenerate the Refund page with the updated content from this PR.
3. Back on WP Admin > Pages, click the link to preview the "Refund and Returns Policy" page. The "Late or missing refunds" and "Sale items" headings should now be aligned correctly.



### Changelog entry

Fixed misaligned elements in generated refunds page issue, updated tags.


<details>

#### Significance




#### Type




#### Message 

Fix alignment issues in the generated content of the Refunds page

#### Comment 

</details>
